packages/amplify_api/android/src/main/kotlin/com/amazonaws/amplify/amplify_api/FlutterApiRequest.kt

```diff
@@ -30,16 +30,17 @@ object FlutterApiRequest {
     private const val BODY_KEY = "body"
     private const val QUERY_PARAM_KEY = "queryParameters"
     private const val HEADERS_KEY = "headers"
+    private val TABS_OUTSIDE_QUOTES_PATTERN = Regex("\t(?=(?:[^'\"`]*(['\"`])[^'\"`]*\\1)*[^'\"`]*\$)")
```

Comment:
Can you add a comment with the unescaped version, or use a raw string?

\t(?=(?:[^'"`]*(['"`])[^'"`]*\1)*[^'"`]*$)

Reply:
Also a small comment as to how it works?

Reply:
Sure, will update.